documentation/Apple Unified Logging and Activity Tracing formats.asciidoc

```diff
@@ -643,30 +721,32 @@ The Shared-Cache Strings (dsc) file header is 16 bytes of size and consists of:
 | 0 | 4 | "hcsd" | Signature
 | 4 | 2 | 1 | Format major version
 | 6 | 2 | 1 | Format minor version
-| 8 | 4 | | [yellow-background]*Unknown (range count)*
-| 12 | 4 | | [yellow-background]*Unknown (uuid count)*
+| 8 | 4 | | Range count
+| 12 | 4 | | Uuid count
 |===
 
+The header is immediately followed by Range descriptors and UUID descriptors.
+
 A range descriptor is 16 bytes of size and consist of:
 
 [cols="1,1,1,5",options="header"]
 |===
 | Offset | Size | Value | Description
-| 0 | 4 | | [yellow-background]*Unknown*
-| 4 | 4 | | Range start (offset?)
-| 8 | 4 | | Range size
-| 12 | 4 | | [yellow-background]*Unknown*
+| 0 | 4 | | UUID descriptor index
+| 4 | 4 | | Range start (offset)
+| 8 | 4 | | Data offset (absolute)
```

Comment:
With "absolute" here I assume you mean "The offset is relative to the start of the file"